lib/session/session.go

```diff
@@ -74,6 +74,8 @@ func NewID() ID {
 // Session is an interactive collaboration session that represents one
 // or many SSH session started by teleport user
```

Comment:
```suggestion
// or many SSH or Kubernetes sessions started by the teleport user
```

Reply:
modified slightly differently since this will probably be used for other types of sessions (maybe)